drivers/cc110x/cc110x_chanmaps.c

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
In a C file they will generate only one symbol and only occupy ROM once. In a header I could make them `static` to prevent the symbol conflicts, but still the same channel map might be stored more than once in ROM.

Reply:
No idea about that. Did you try compare both alternatives in ROM consumtion?

Reply:
Just did. 20 bytes more for moving it to a different .c file and dropping the `static` qualifier.

Current state:
```
   text    data     bss     dec     hex filename
 128048     552   97752  226352   37430 /home/maribu/Repos/software/RIOT/examples/gnrc_networking/bin/msba2/gnrc_networking.elf
```

After moving it to `cc110x.c` and dropping the `static` qualifier:
```
   text    data     bss     dec     hex filename
 128068     552   97752  226372   37444 /home/maribu/Repos/software/RIOT/examples/gnrc_networking/bin/msba2/gnrc_networking.elf
```